docs/detections/prebuilt-rules/rule-details/unusual-parent-child-relationship.asciidoc

```diff
@@ -23,7 +23,7 @@ Identifies Windows programs run from unexpected parent processes. This could ind
 
 *References*:
 
-* https://github.com/sbousseaden/Slides/blob/master/Hunting MindMaps/PNG/Windows Processes TH.map.png
+* https://github.com/sbousseaden/Slides/blob/master/Hunting MindMaps/PNG/Windows Processes%20TH.map.png
```

Comment:
```suggestion
* https://github.com/sbousseaden/Slides/blob/master/Hunting%20MindMaps/PNG/Windows%20Processes%20TH.map.png
```

Reply:
thanks - we can make this change in the repo, to reflect next time